dts/arm/nxp/nxp_rt7xx_ns.dtsi

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+
+/ {
+	soc {
+		sram: sram@0 {
+			ranges = <0x0 0x0 0x780000
+				0x20000000 0x20000000 0x780000>;
+		};
+
+		peripheral: peripheral@40000000 {
+			ranges = <0x0 0x40000000 0x10000000>;
+		};
+	};
+};
+
+#if defined SOC_MIMXRT798S_CPU0
+#include "nxp_rt7xx_cpu0.dtsi"
```

Comment:
cpu0 doesn't really tell us much. Can you be more specific.

Reply:
cpu0 is used in RT700 to refer to cm33 cpu0, so I think there is no need to update.

Reply:
What about `CM33_CPU0` and `cm33_cpu0`?